src/components/evm/chains.jsx

```diff
@@ -1,119 +1,164 @@
-import evm_chains from "../../data/evm_chains.json";
-import gas_services from "../../data/gas_services.json";
-import gateways from "../../data/gateways.json";
+import React, { useState, useEffect } from "react";
 import { ellipse } from "../../utils";
 import Copy from "../copy";
 import AddChain from "../web3";
 
+function toHex(decimal) {
+  return decimal.toString(16).toUpperCase();
+}
+
 export default ({ environment = "mainnet" }) => {
-  const _evm_chains = evm_chains?.[environment] || [];
-  const _gateways = gateways?.[environment] || [];
-  const _gas_services = gas_services?.[environment] || [];
+  const [chainConfigs, setChainConfigs] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
(On testnet only) The L2 testnets seem to be missing their logos. For example, 
<img width="756" alt="missing_logos" src="https://github.com/user-attachments/assets/3154b0fe-10e9-4006-bc75-827111777ed0">